docs/security/security-mtls.md

```diff
@@ -0,0 +1,67 @@
+# Security: Mutual TLS
+
+
+Mutual TLS is enabled in a single step, which can be further automated to allow for zero-config security. 
+
+
+The important element to enable mTLS is the identity of the workload, and it is enabled with the help of pluggable solutions like SPIFFE and cert-manager. Identity managemement (including cert provisioning/attestation, hourly rotation and validation) is completely automated. For mTLS, peer certificates are automatically validated as governed by policy.
+
+
+## Architecture
+
+The key ingredients for zero-trust security with mTLS are Identity, AuthN/Z, Encryption and Audit. These are all enabled in a single step, which can be further automated for zero-config security which follows the workload instantly across any platform, and independent of infrastructure. Enforcement is consistent across all appEdge bindings, including native/proxyless HTTP apps.
+
+* Identity: enabled with the help of pluggable solutions like SPIFFE and cert-manager. Identity managemement (including cert provisioning/attestation, hourly rotation and validation) is completely automated.
```

Comment:
suggestion: Put Identity/AuthN/Z... in bold

Reply:
Is bold alread?